leader/leader.go

```diff
@@ -167,6 +167,7 @@ func Become(ctx context.Context, lockName string, opts ...Option) error {
 				log.Info("Leader lock configmap owner reference must be a pod.", "OwnerReference", existingOwners[0])
 			default:
 				leaderPod := &corev1.Pod{}
+				// leaderNode := &corev1.Node{}
```

Comment:
we should not merge  code commented

Reply:
thanks, I will delete the comment at the new commit.